src/content/concepts/configuration.md

```diff
@@ -5,11 +5,12 @@ contributors:
   - TheLarkInn
   - simon04
   - EugeneHlushko
+  - byzyk
 ---
 
-You may have noticed that few webpack configurations look exactly alike. This is because **webpack's configuration file is a JavaScript file that exports an object.** This object is then processed by webpack based upon its defined properties.
+You may have noticed that few webpack configurations look exactly alike. This is because __webpack's configuration file is a JavaScript file that exports an object.__ This object is then processed by webpack based upon its defined properties.
```

Comment:
later on we mention that configuration is an object or a function please also mention it here in the `__` section

Reply:
I don't think it's necessary to mention it as it's covered in more details [here](https://webpack.js.org/configuration/configuration-types/#exporting-multiple-configurations). You can export a function, Promise or array but in the end, it's just a plain object returned. 

Maybe we can add it as a note below, saying something like `webpack configuration is always presented as an object but can be exported as an array, promise, function etc + include the link I shared above`. What do you think?

Reply:
Maybe change it to `... that exports a webpack [configuration](/configuration/)` because `javascript file that exports ....` is not about how its presented but about what it exports explicitly

Reply:
I guess the idea here was to show that webpack config is just a plain JS object and you can mess up with it in any way you like. Changing it to `exports webpack config` might lead to confusion, especially among beginners.

Reply:
Then there is more advanced guide on how to export function or array etc. and we can add that one as a note here, this will allow us to keep the existing guide more beginner friendly and have a reference that exporting an object is not the only option. Does it make sense?

Reply:
dunno man its plain straight to me that we can export w/e from javascript webpack config file, lets hear from more folks then

CC @jeremenichelli @montogeek

Reply:
on the other hand, the text was like this before (its not like you've updated it, just the syntax around it). we can change it later if more folks agree on it